docker/Dockerfile

```diff
@@ -62,6 +62,10 @@ ENV PIP_NO_CACHE_DIR=off \
 COPY /dist/requirements.txt /tmp/dist/requirements.txt
 RUN set -x \
   && buildDeps="" \
+  # ability to install git dependencies
+  && buildDeps="$buildDeps \
+  git \
```

Comment:
We won't be shipping `git` in prod images ever. If you ever need it, it should probably be in the builder image: https://github.com/getsentry/sentry/blob/master/docker/builder.dockerfile

Reply:
fwiw this is installed and uninstalled within the same build step, as all buildDeps.

Reply:
oh, right missed that. Still it would be ideal not to use git. Will propose something below.